x-pack/legacy/plugins/searchprofiler/public/editor/worker/worker.d.ts

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+declare module '!!raw-loader!./worker.js';
```

Comment:
This is to satsify TS's constraint that all imported modules be declared (the lang server would complain in the neighbouring `./index.ts` file that it has no idea what `!!raw-loader!./worker.js` is.) TS doesn't really care much about the content of the string here if I'm not mistaken. We could have actually said `decalre module '!!raw-loader!*` to cover all cases where we'd want to load JS/TS as a string but because this is living in a deeply nested dir I made the declaration specific. The `!!raw-loader!` bit is for [webpack](https://webpack.js.org/loaders/raw-loader/#examples) :)